examples/llm-with-simple-requests/__main__.py

```diff
@@ -0,0 +1,53 @@
+import requests
+import os
+
+from damavand.cloud.aws.controllers.llm import AwsLlmController
+
+
+controller = AwsLlmController(
+    name="my-dspy",
+    region="eu-west-1",
+)
+
+
+def ask(question: str, role: str) -> None:
```

Comment:
How would the user know how the request should be structured? In this case, it assumes that OpenAI is used behind the scenes. What if a different LLM is there?

Reply:
This is OpenAI ChatCompletion API Model which we are using as standard. All the models that we are providing are supporting this model of API. It is independent of the model.

Reply:
add to docsrtings